templates/typesetting/elements/typesetter.html

```diff
@@ -26,14 +26,56 @@ <h4>Typesetter: {{ round.typesettingassignment.typesetter.full_name }} <a onclic
             <td>{{ round.typesettingassignment.status|capfirst }}</td>
         </tr>
     </table>
+</div>
+
+{% for proofing in round.galleyproofing_set.all %}
+    <div class="callout breakdown file">
+        <div class="callout-title">
+            <small><a class="pull-right" href="{% url 'admin:typesetting_galleyproofing_change' proofing.pk %}" target="_blank">Edit in Admin</a></small>
+            <h4>Proofreader #{{ forloop.counter }}: {{ proofing.proofreader.full_name }} <a onclick="return popitup('{% url 'send_user_email_article' proofing.proofreader.pk article.pk %}')"><i class="fa fa-envelope">&nbsp;</i></a></h4>
+        </div>
+        <table class="table small">
+            <tr>
+                <th>ID</th>
+                <th>Created</th>
+                <th>Manager</th>
+                <th>Notified</th>
+                <th>Due</th>
+                <th>Completed</th>
+                <th>Status</th>
+                <th></th>
+            </tr>
+            <tr>
+                <td>{{ proofing.pk }}</td>
+                <td>{{ proofing.assigned }}</td>
+                <td>{{ proofing.manager.full_name }}</td>
+                <td>{{ proofing.notified|bool_fa }}</td>
+                <td>{{ proofing.due }}</td>
+                <td>{% if proofing.completed %}{{ proofing.completed }}{% else %}{{ proofing.completed|bool_fa }}{% endif %}</td>
+                <td>{{ proofing.status|capfirst }}</td>
+                <td><a href="{% url 'typesetting_manage_proofing_assignment' article.pk proofing.pk %}">Manage</a></td>
+            </tr>
+        </table>
+    </div>
+{% endfor %}
+
+<div class="callout bs-callout-warning">
+    {% if not round.has_completed_proofing %}
+    <span class="fa fa-check-circle"></span> {{ round.typesettingassignment.friendly_status }}<br/><br/>
+    {% else %}
+        <span class="fa fa-exclamation-triangle"> {{ round.has_completed_proofing.count }} out of {{ round.galleyproofing_set.count }} proofing tasks completed.<br/><br/></span>
+    {% endif %}
+    <div class="button-group">
+    {% if round.has_completed_proofing %}
+        <a href="" class="alert button">Request Corrections</a>
+    {% endif %}
+    {% if round.typesettingassignment.reviewed and round.typesettingassignment.review_decision == 'proofing' %}
+        <a href="{% url 'typesetting_assign_proofreader' article.pk %}" class="warning button">Assign{% if round.galleyproofing_set.all %} More{% endif %} Proofreaders</a>
+    {% elif round.typesettingassignment.reviewed and round.typesettingassignment.review_decision == 'corrections' %}
+        <a href="" class="success button">Start New Typesetting Round</a>
+    {% endif %}
+    <a href="{% url 'typesetting_review_assignment' article.pk round.typesettingassignment.pk %}" class="success button">Review Typesetting</a>
```

Comment:
As discussed, move the proofing bit to its own component. Same with the buttons

Reply:
Done! Simples!